### ISSUE TITLE
'conan info' command not working when the current directory contains conanbuildinfo.txt

### ISSUE THREAD

**Issue by hamparawa** [conan-io/conan#2202]
When `conan info` is invoked in a build directory where a previous `conan install` command is invoked, the options for the target package are passed incorrectly it seems.

Suppose I have two packges,
`PackageA@0.1@demo/testing` with options `OptionA1` and `OptionA2`
`PackageB@0.1@demo/testing` with options `OptionB1` and `OptionB2`

And in the directory of PackageB recipe,

```
mkdir build
cd build
conan install ..
conan info PackageA@0.1@demo/testing
```
Returns following error.

`
ERROR: PackageA@0.1@demo/testing: 'options.OptionB1' doesn't exist
Possible options are ['OptionA1', 'OptionA2']
`

However, when the generated `conanbuildinfo.txt` in the build directory is removed, the command executes fine.

I'm using Conan version 0.30.3

**Comment by memsharded** [comment]
This seems a bug, thanks very much for reporting.
I am having a look, lets see if we can fix for the 1.0-beta release.

**Comment by memsharded** [comment]
Released in conan 1.0.0-beta:  http://docs.conan.io/en/latest/conan1.0.html 
Please try and report if any problem is found. Thanks very much!
